config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: mercari/spanner-autoscaler
-  newTag: latest
+  newTag: local
```

Comment:
This was changed by mistake, during local testing. I will revert this in the next PR.